frontend/src/pages/AccountSent.vue

```diff
@@ -12,61 +12,88 @@
       </div>
     </div>
     <div v-else class="q-mx-auto" style="max-width: 800px">
-      <account-sent-table :sendMetadata="sendMetadata" />
+      <div v-if="needsSignature" class="text-center q-mb-md">
```

Comment:
Why is the html here an if/else, where the else has it's own if/else-if/else? Should we just flatten that?